pages/code-of-conduct.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page-fullwidth
+title: "HPC Carpentry Code of Conduct"
+permalink: /code-of-conduct/
+---
+
+We ask that people adhere to the guidelines of
```

Comment:
Let's provide an overview of our Code. Suggestion follows.

---

We are dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity.  In order to foster a positive and professional learning environment we encourage the following practices:

* Use welcoming and inclusive language.
* Be respectful of different viewpoints and experiences.
* Gracefully accept constructive criticism.
* Focus on what is best for the community.
* Show courtesy and respect towards other community members.

Our community has chosen to adopt the official [Carpentries Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html). If you have questions about the policy, concerns over observed behaviours, or need to report a violation of the Code, we ask that you reach out to any member of the [HPC Carpentry Steering Committee](/contact#steering). Your communications will be held in confidence. We pledge to acknowledge your message within 24 hours, and to take action within 30 calendar days.

Reply:
Thanks